tfx/examples/cifar10/README.md

```diff
@@ -0,0 +1,88 @@
+
+# CIFAR-10 Transfer Learning and MLKit integration Example
+This example illustrates how to use Transfer Learning for image classification with TFX, and use trained model to do object detection with [MLKit](https://developers.google.com/ml-kit)
+
+## Instruction
+
+Create a Python 3 virtual environment for this example and activate the
+`virtualenv`:
+
+```
+virtualenv -p python3.7 cifar10
+source ./cifar10/bin/activate
+```
+
+Next, install the dependencies required by the CIFAR-10 example (appropriate
+version of TF2 will be installed automatically).
+
+```
+pip install tfx
+pip install tflite-support
```

Comment:
Use a range for dependencies.

nit: put all pip install commands on one line (which helps the silly pip dependency management)